src/H5Ppublic.h

```diff
@@ -3730,7 +3730,7 @@ H5_DLL herr_t H5Pget_file_image(hid_t fapl_id, void **buf_ptr_ptr, size_t *buf_l
  *          The callbacks must have been previously set with
  *          H5Pset_file_image_callbacks() in the file access property list.
  *
- *          Upon the successful return of H5Pget_file_image_callbacks(), the
+ *          Upon the successful return of H5Pset_file_image_callbacks(), the
```

Comment:
I think the original text was correct here (not because I specifically know how the code works, but what makes sense from the context).  It seems to me that if H5Pget_file_image_callbacks() is retrieving the callbacks, the struct will contain what H5Pget_file_image_callbacks() retrieved when it returns successfully, and H5Pset_file_image_callbacks() is not returning here.

Reply:
reverted (PR merge issue)